loop/impact scripts/Analysis.py

```diff
@@ -0,0 +1,94 @@
+import pandas as pd
```

Comment:
remove impact scripts files. Already integrated in code

Reply:
Removed.